modules/kubernetes-addons/csi-secrets-store-provider-aws/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "kubernetes_namespace" "csi_secrets_store_provider_aws" {
```

Comment:
I believe we can remove this and instead use whats provided in the `helm_addon` sub-module https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/deec7d5caea47c06dea48fa616ad2c56e52c3cce/modules/kubernetes-addons/helm-addon/main.tf#L64

Reply:
Or alternatively set `create_kubernetes_namespace` and `create_kubernetes_service_account` to `false` in the `module helm_addon` call.